### ISSUE TITLE
Getting java.util.MissingFormatArgumentException After Clicking stop video record button. 

### ISSUE THREAD

**Issue by pingothedoer** [afollestad/material-camera#64]
`private void recordVideo(String fileName){`
	`File saveFoler = new File(fileName);`
	`if (!saveFoler.exists() || ! saveFoler.isDirectory()){`
		`saveFoler.mkdirs();`
	`}`

	new MaterialCamera(this)
			.allowRetry(true)
			.autoSubmit(false) 
			.saveDir(saveFoler)
			.start(CAMERA_RQ);
}

###### LogCat 

```Gradle
E/AndroidRuntime: FATAL EXCEPTION: main
				  Process: com.xyz.xyz, PID: 9203
				  java.util.MissingFormatArgumentException: Format specifier: 20v
					  at java.util.Formatter.getArgument(Formatter.java:1111)
					  at java.util.Formatter.doFormat(Formatter.java:1076)
					  at java.util.Formatter.format(Formatter.java:1042)
					  at java.util.Formatter.format(Formatter.java:1011)
					  at java.lang.String.format(String.java:1803)
					  at java.lang.String.format(String.java:1777)
					  at com.afollestad.easyvideoplayer.EasyVideoPlayer.LOG(EasyVideoPlayer.java:833)
					  at com.afollestad.easyvideoplayer.EasyVideoPlayer.prepare(EasyVideoPlayer.java:361)
					  at com.afollestad.easyvideoplayer.EasyVideoPlayer.onSurfaceTextureAvailable(EasyVideoPlayer.java:558)
					  at android.view.TextureView.getHardwareLayer(TextureView.java:370)
					  at android.view.View.updateDisplayListIfDirty(View.java:15373)
					  at android.view.View.getDisplayList(View.java:15418)
					  at android.view.View.draw(View.java:16188)
					  at android.view.ViewGroup.drawChild(ViewGroup.java:3709)
					  at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3502)
					  at android.view.View.draw(View.java:16469)
					  at android.widget.FrameLayout.draw(FrameLayout.java:598)
					  at android.view.View.updateDisplayListIfDirty(View.java:15396)
					  at android.view.View.getDisplayList(View.java:15418)
					  at android.view.View.draw(View.java:16188)
					  at android.view.ViewGroup.drawChild(ViewGroup.java:3709)
					  at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3502)
					  at android.view.View.draw(View.java:16469)
					  at android.widget.FrameLayout.draw(FrameLayout.java:598)
					  at android.view.View.updateDisplayListIfDirty(View.java:15396)
					  at android.view.View.getDisplayList(View.java:15418)
					  at android.view.View.draw(View.java:16188)
					  at android.view.ViewGroup.drawChild(ViewGroup.java:3709)
					  at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3502)
					  at android.view.View.updateDisplayListIfDirty(View.java:15391)
					  at android.view.View.getDisplayList(View.java:15418)
					  at android.view.View.draw(View.java:16188)
					  at android.view.ViewGroup.drawChild(ViewGroup.java:3709)
					  at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3502)
					  at android.view.View.updateDisplayListIfDirty(View.java:15391)
					  at android.view.View.getDisplayList(View.java:15418)
					  at android.view.View.draw(View.java:16188)
					  at android.view.ViewGroup.drawChild(ViewGroup.java:3709)
					  at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3502)
					  at android.view.View.updateDisplayListIfDirty(View.java:15391)
					  at android.view.View.getDisplayList(View.java:15418)
					  at android.view.View.draw(View.java:16188)
					  at android.view.ViewGroup.drawChild(ViewGroup.java:3709)
					  at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3502)
					  at android.view.View.updateDisplayListIfDirty(View.java:15391)
					  at android.view.View.getDisplayList(View.java:15418)
					  at android.view.View.draw(View.java:16188)
					  at android.view.ViewGroup.drawChild(ViewGroup.java:3709)
					  at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3502)
					  at android.view.View.draw(View.java:16469)
					  at android.widget.FrameLayout.draw(FrameLayout.java:598)
					  at com.android.internal.policy.impl.PhoneWindow$DecorView.draw(PhoneWindow.java:3107)
					  at android.view.View.updateDisplayListIfDirty(View.java:15396)
					  at android.view.View.getDisplayList(View.java:15418)
					  at android.view.ThreadedRenderer.updateViewTreeDisplayList(ThreadedRenderer.java:310)
					  at android.view.ThreadedRenderer.updateRootDisplayList(ThreadedRenderer.java:316)
					  at android.view.ThreadedRenderer.draw(ThreadedRenderer.java:355)
					  at android.view.ViewRootImpl.draw(ViewRootImpl.java:2890)
					  at android.view.ViewRootImpl.performDraw(ViewRootImpl.java:2684)
					  at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2271)
					  at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1264)
					  at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6943)
					  at android.view.Choreographer$CallbackRecord.run(Choreographer.java:777)
					  at android.view.Choreographer.doCallbacks(Choreographer.java:590)
					  at android.view.Choreographer.doFrame(Choreographer.java:560)
					at android.view.Choreographe
```

###### Include the following:
 - Material Camera version: `0.2.13`
 - Device OS version: `5.1`
 - Device Manufacturer: `Samsun`
 - Device Name: `Galaxy S6`

**Comment by pingothedoer** [comment]
@afollestad can u plx take a look at it.

**Comment by afollestad** [comment]
I get notified by email of new issues without you mentioned me just so you know :wink:

---

The log shows it's actually in [Easy Video Player](https://github.com/afollestad/easy-video-player), not this library. But the problem is related to the URI you're loading.

However, it should never happen if you're using the latest version of this library.

**Comment by pingothedoer** [comment]
I am using the latest one, but if its URI related issue, at least it should not make my app crash. :-(

**Comment by ibrahimGhailani** [comment]
I'm facing the same issue. Clicking the stop button when device is set to RTL language causes a FC.

```groovy 
E/AndroidRuntime: FATAL EXCEPTION: main
                                                                  Process: , PID: 5548
java.util.MissingFormatArgumentException: Format specifier: D
                                                                      at java.util.Formatter.getArgument(Formatter.java:1111)
                                                                      at java.util.Formatter.doFormat(Formatter.java:1076)
                                                                      at java.util.Formatter.format(Formatter.java:1042)
                                                                      at java.util.Formatter.format(Formatter.java:1011)
                                                                      at java.lang.String.format(String.java:1554)
                                                                      at java.lang.String.format(String.java:1528)
                                                                      at com.afollestad.easyvideoplayer.EasyVideoPlayer.LOG(EasyVideoPlayer.java:833)
                                                                      at com.afollestad.easyvideoplayer.EasyVideoPlayer.prepare(EasyVideoPlayer.java:361)
                                                                      at com.afollestad.easyvideoplayer.EasyVideoPlayer.onSurfaceTextureAvailable(EasyVideoPlayer.java:558)
                                                                      at android.view.TextureView.getHardwareLayer(TextureView.java:368)
                                                                      at android.view.View.updateDisplayListIfDirty(View.java:15157)
                                                                      at android.view.View.draw(View.java:15954)
                                                                      at android.view.ViewGroup.drawChild(ViewGroup.java:3609)
                                                                      at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3399)
                                                                      at android.view.View.draw(View.java:16187)
                                                                      at android.view.View.updateDisplayListIfDirty(View.java:15180)
                                                                      at android.view.View.draw(View.java:15954)
                                                                      at android.view.ViewGroup.drawChild(ViewGroup.java:3609)
                                                                      at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3399)
                                                                      at android.view.View.draw(View.java:16187)
                                                                      at android.view.View.updateDisplayListIfDirty(View.java:15180)
                                                                      at android.view.View.draw(View.java:15954)
                                                                      at android.view.ViewGroup.drawChild(ViewGroup.java:3609)
                                                                      at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3399)
                                                                      at android.view.View.updateDisplayListIfDirty(View.java:15175)
                                                                      at android.view.View.draw(View.java:15954)
                                                                      at android.view.ViewGroup.drawChild(ViewGroup.java:3609)
                                                                      at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3399)
                                                                      at android.view.View.updateDisplayListIfDirty(View.java:15175)
                                                                      at android.view.View.draw(View.java:15954)
                                                                      at android.view.ViewGroup.drawChild(ViewGroup.java:3609)
                                                                      at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3399)
                                                                      at android.view.View.updateDisplayListIfDirty(View.java:15175)
                                                                      at android.view.View.draw(View.java:15954)
                                                                      at android.view.ViewGroup.drawChild(ViewGroup.java:3609)
                                                                      at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3399)
                                                                      at android.view.View.updateDisplayListIfDirty(View.java:15175)
                                                                      at android.view.View.draw(View.java:15954)
                                                                      at android.view.ViewGroup.drawChild(ViewGroup.java:3609)
                                                                      at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3399)
                                                                      at android.view.View.draw(View.java:16187)
                                                                      at com.android.internal.policy.PhoneWindow$DecorView.draw(PhoneWindow.java:2690)
                                                                      at android.view.View.updateDisplayListIfDirty(View.java:15180)
                                                                      at android.view.ThreadedRenderer.updateViewTreeDisplayList(ThreadedRenderer.java:281)
                                                                      at android.view.ThreadedRenderer.updateRootDisplayList(ThreadedRenderer.java:287)
                                                                      at android.view.ThreadedRenderer.draw(ThreadedRenderer.java:322)
                                                                      at android.view.ViewRootImpl.draw(ViewRootImpl.java:2623)
                                                                      at android.view.ViewRootImpl.performDraw(ViewRootImpl.java:2442)
                                                                      at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2075)
                                                                      at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1115)
                                                                      at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6023)
                                                                      at android.view.Choreographer$CallbackRecord.run(Choreographer.java:858)
                                                                      at android.view.Choreographer.doCallbacks(Choreographer.java:670)
                                                                      at android.view.Choreographer.doFrame(Choreographer.java:606)
                                                                      at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:844)
                                                                      at android.os.Handler.handleCallback(Handler.java:739)
                                                                      at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                      at android.os.Looper.loop(Looper.java:148)
                                                                      at android.app.ActivityThread.main(ActivityThread.java:5422)
                                                                      at java.lang.reflect.Method.invoke(Native Method)
                                                                      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
                                                                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```
Material Camera: 0.2.14
LG Nexus 5X (6.0.1) and Moto G (5.1)

**Comment by FaaaX** [comment]
I'm facing the same issue. Clicking the stop button. I'm using the newest version. 

`E/Report ::: java.util.MissingFormatArgumentException: Format specifier: 20S
                                                                 
                                                                 --------- Stack trace ---------
                                                                 
                                                                     at java.util.Formatter.getArgument(Formatter.java:1111)
                                                                     at java.util.Formatter.doFormat(Formatter.java:1076)
                                                                     at java.util.Formatter.format(Formatter.java:1042)
                                                                     at java.util.Formatter.format(Formatter.java:1011)
                                                                     at java.lang.String.format(String.java:1554)
                                                                     at java.lang.String.format(String.java:1528)
                                                                     at com.afollestad.easyvideoplayer.EasyVideoPlayer.LOG(EasyVideoPlayer.java:833)
                                                                     at com.afollestad.easyvideoplayer.EasyVideoPlayer.prepare(EasyVideoPlayer.java:361)
                                                                     at com.afollestad.easyvideoplayer.EasyVideoPlayer.onSurfaceTextureAvailable(EasyVideoPlayer.java:558)
                                                                     at android.view.TextureView.getHardwareLayer(TextureView.java:368)
                                                                     at android.view.View.updateDisplayListIfDirty(View.java:15173)
                                                                     at android.view.View.draw(View.java:15969)
                                                                     at android.view.ViewGroup.drawChild(ViewGroup.java:3612)
                                                                     at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3402)
                                                                     at android.view.View.draw(View.java:16202)
                                                                     at android.view.View.updateDisplayListIfDirty(View.java:15196)
                                                                     at android.view.View.draw(View.java:15969)
                                                                     at android.view.ViewGroup.drawChild(ViewGroup.java:3612)
                                                                     at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3402)
                                                                     at android.view.View.draw(View.java:16202)
                                                                     at android.view.View.updateDisplayListIfDirty(View.java:15196)
                                                                     at android.view.View.draw(View.java:15969)
                                                                     at android.view.ViewGroup.drawChild(ViewGroup.java:3612)
                                                                     at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3402)
                                                                     at android.view.View.updateDisplayListIfDirty(View.java:15191)
                                                                     at android.view.View.draw(View.java:15969)
                                                                     at android.view.ViewGroup.drawChild(ViewGroup.java:3612)
                                                                     at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3402)
                                                                     at android.view.View.updateDisplayListIfDirty(View.java:15191)
                                                                     at android.view.View.draw(View.java:15969)
                                                                     at android.view.ViewGroup.drawChild(ViewGroup.java:3612)
                                                                     at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3402)
                                                                     at android.view.View.updateDisplayListIfDirty(View.java:15191)
                                                                     at android.view.View.draw(View.java:15969)
                                                                     at android.view.ViewGroup.drawChild(ViewGroup.java:3612)
                                                                     at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3402)
                                                                     at android.view.View.updateDisplayListIfDirty(View.java:15191)
                                                                     at android.view.View.draw(View.java:15969)
                                                                     at android.view.ViewGroup.drawChild(ViewGroup.java:3612)
                                                                     at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3402)
                                                                     at android.view.View.draw(View.java:16202)
                                                                     at com.android.internal.policy.PhoneWindow$DecorView.draw(PhoneWindow.java:2690)
                                                                     at android.view.View.updateDisplayListIfDirty(View.java:15196)
                                                                     at android.view.ThreadedRenderer.updateViewTreeDisplayList(ThreadedRenderer.java:281)
                                                                     at android.view.ThreadedRenderer.updateRootDisplayList(ThreadedRenderer.java:287)
                                                                     at android.view.ThreadedRenderer.draw(ThreadedRenderer.java:322)
                                                                     at android.view.ViewRootImpl.draw(ViewRootImpl.java:2627)
                                                                     at android.view.ViewRootImpl.performDraw(ViewRootImpl.java:2446)
                                                                     at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2079)
                                                                     at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1119)
                                                                     at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6060)
                                                                     at android.view.Choreographer$CallbackRecord.run(Choreographer.java:858)
                                                                     at android.view.Choreographer.doCallbacks(Choreographer.java:670)
                                                                     at android.view.Choreographer.doFrame(Choreographer.java:606)
                                                                     at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:844)
                                                                     at android.os.Handler.handleCallback(Handler.java:746)
                                                                     at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                     at android.os.Looper.loop(Looper.java:148)
                                                                     at android.app.ActivityThread.main(ActivityThread.java:5443)
                                                                     at java.lang.reflect.Method.invoke(Native Method)
                                                                     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:728)
                                                                     at com.android.internal.os.ZygoteInit.main(ZygoteIn`

**Comment by FaaaX** [comment]
Fixed in 0.3.0.
